src/process_utility.c

```diff
@@ -3501,13 +3531,21 @@ process_altertable_start_table(ProcessUtilityArgs *args)
 				if (NULL == ht)
 					process_altertable_chunk_set_tablespace(cmd, relid);
 				break;
+#if PG15_GE
+			case AT_SetAccessMethod:
+				process_set_access_method(cmd, args);
```

Comment:
Will we open the doors for any TAM?? Shouldn't it be restricted to hyperstore??

Reply:
We have always had support for any TAM. See `tableam.sql` test. This just adds an additional setting if you set the `hyperstore` access method.